tasks/test-unit.sh

```diff
@@ -3,16 +3,14 @@
 source /etc/profile.d/chruby.sh
 chruby ruby
 
-set +e
-#create user for ShelloutTypes::File tests
-chroot /tmp/ubuntu-chroot /bin/bash -c 'useradd -G nogroup shellout'
-set -e
+# we need sudo for our chroot operations in the shellout_types tests
+apt install sudo
 
 pushd bosh-linux-stemcell-builder
   bundle install --local
 
   pushd bosh-stemcell
     bundle exec rspec spec/
-    bundle exec rspec spec/ --tag shellout_types
+    OS_IMAGE="$(readlink -f ../../os-image-tarball/*.tgz)" bundle exec rspec spec/ --tag shellout_types
```

Comment:
Does this change depend on other changes made earlier to the bionic branch? Or would this also work on the xenial branch?

Reply:
yes it could break xenial. but i think the xenial shellout types where broken anyways
these commits should be backported to xenial
https://github.com/cloudfoundry/bosh-linux-stemcell-builder/commit/0d847cb4a71669ba89219aae30e68f8bfa1ca4c4#diff-5b8ef45c27bd5967af777cf9099a2f2aa21462b8e99f13af5cc97eb03e0dce7a
https://github.com/cloudfoundry/bosh-linux-stemcell-builder/commit/7b61525be00febe180b354b93a938b99a5fb7f3c#diff-5b8ef45c27bd5967af777cf9099a2f2aa21462b8e99f13af5cc97eb03e0dce7a